src/commands/function/build.ts

```diff
@@ -24,6 +24,10 @@ const createManifest = (
   return manifest;
 };
 
+const renameWasm = (path: string, oldName: string, newName: string) => {
+  execSync(`mv ${oldName} ${newName}`, { cwd: path, stdio: "inherit" });
+};
+
 export const run = (options: {
   debug: boolean;
   name: string;
```

Comment:
So, yeah....this renaming became obsolete when I made my (failing) stab at letting init handle it via modifying ascript.json, which should now be fixed by https://github.com/txlabs/blockless-cli/pull/30 so it can be dropped like a bad habit